### PR TITLE
Fixed Design Issues - on find/search Page

### PR DESCRIPTION
**Pull request by YugaaySolutions**
Design Fixes for the find/search page.

Border around the search options.
Focus on the input and the button

### CODE REVIEW

**Comment by JayyajGH** [comment]
I'll move my comments across from the closed PR to this one...

- All colours should be defined in the _colour_vars.scss file and referenced from here. Can existing colours be used rather than creating new ones?
- Does box-shadow need the webkit prefix? I believe this is now supported everywhere.
- Does the ref-div:focus class need to be so specific with the extra div?
- If activeClass always returns "search_op" then why not just list it on the element itself?
- Can the class name better reflect what it does rather than "ref-div". That can confuse it with the name of the reference.
- Where is focus-class used?

File: components/Autocomplete.vue [hunk 1]
```diff
@@ -70,10 +70,15 @@
 
 /* iteminp class is passed into this component in a prop */
 .iteminp ul {
+  /*
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Whats the purpose of this? Are we trying to comment this code out? If so, remove it, if not remove the /*

File: components/Autocomplete.vue [hunk 2]
```diff
@@ -70,10 +70,15 @@
...
 /* iteminp class is passed into this component in a prop */
 .iteminp ul {
+  /* 
   width: 100% !important;
   right: 0px !important;
-  padding-right: 15px !important;
-  padding-left: 15px !important;
+  margin-right: 15px !important;
+  margin-left: 15px !important;
+  */
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Same as above, this is just noise

File: components/Autocomplete.vue [hunk 3]
```diff
@@ -70,10 +70,15 @@
 
 /* iteminp class is passed into this component in a prop */
 .iteminp ul {
+  /* 
   width: 100% !important;
   right: 0px !important;
-  padding-right: 15px !important;
-  padding-left: 15px !important;
+  margin-right: 15px !important;
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Can we avoid important?

File: components/Autocomplete.vue [hunk 4]
```diff
@@ -70,10 +70,15 @@
...
+  margin-left: 15px !important;
+  */
+  /*Width adjusted for the border of the search options*/
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Lets try and keep comments consistent, IE space after the first * and before the last *

File: components/Autocomplete.vue [hunk 5]
```diff
@@ -70,10 +70,15 @@
...
+  width: 96% !important;
+  right: 13px !important;
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Can we avoid the important?

File: components/Autocomplete.vue [hunk 6]
```diff
@@ -70,10 +70,15 @@
...
+  /* 
   width: 100% !important;
   right: 0px !important;
-  padding-right: 15px !important;
-  padding-left: 15px !important;
+  margin-right: 15px !important;
+  margin-left: 15px !important;
+  */
+  /*Width adjusted for the border of the search options*/
+  width: 96% !important;
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Can we avoid the important?

File: components/Autocomplete.vue [hunk 7]
```diff
@@ -159,6 +164,28 @@
   color: white;
 }
 
+
+/*Style for border round the search options*/
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Same as above, lets keep comments consistent

File: components/Autocomplete.vue [hunk 8]
```diff
@@ -159,6 +164,28 @@
...
+.search_op {
+  border: 1px solid #61ae25;
+  border-radius: 0.2rem;
+}
+
+div .ref-div:focus {
+  background-color: #fff;
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Same as above re: variables

File: components/Autocomplete.vue [hunk 9]
```diff
@@ -159,6 +164,28 @@
...
+  border-color: #80bdff;
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Same as above re: variables

File: components/Autocomplete.vue [hunk 10]
```diff
@@ -159,6 +164,28 @@
...
+  outline: 0;
+  border-radius: 0.2rem;
+  box-shadow: 0 0 0 0.2rem rgba(0, 123, 255, 0.25);
```
Comment (aaronferris) [inline components/Autocomplete.vue]: is this box shadow colour re-usable? Worth a variable?

File: components/Autocomplete.vue [hunk 11]
```diff
@@ -159,6 +164,28 @@
...
+}
+
+.focus-class:focus {
+  border-color: #ced4da;
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Same as above re: variables

File: components/Autocomplete.vue [hunk 12]
```diff
@@ -159,6 +164,28 @@
...
+  border-right: none;
+  -webkit-box-shadow: none;
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Is this needed? I was under the impression autoprefixer was working out of the box with next.js, if not I would recommend we have a task to add it. Adding prefixes will become tedious.

File: pages/find/search/_term.vue [hunk 13]
```diff
@@ -167,7 +167,8 @@ export default {
     this.$store.dispatch('messages/clear')
 
     // Focus on field to grab their attention.
-    this.$refs.autocomplete.$refs.input.focus()
+    //this.$refs.autocomplete.$refs.input.focus()
```
Comment (aaronferris) [inline pages/find/search/_term.vue]: Remove commented code, if we don't need it it's just noise.

File: components/Autocomplete.vue [hunk 14]
```diff
@@ -479,8 +506,8 @@ export default {
       this.focusList = i
     },
 
-    activeClass(i) {
-      const focusClass = i === this.focusList ? 'focus-list' : ''
+     activeClass(i) {
```
Comment (aaronferris) [inline components/Autocomplete.vue]: Odd indentation

**Comment by aaronferris** [comment]
@YugaaySolutions  I have added some comments inline in the diff, apart from that welcome to the project and thanks for your contribution :)

**Pull request closed without merging**